bin/install-qa-check.d/60openrc

```diff
@@ -33,6 +33,23 @@ openrc_check() {
 			done
 		done
 	fi
+
+	# Make sure deprecated #!/sbin/runscript isn't being used:
+	if [[ ! -d "${ED}"/etc/init.d ]] ; then
+		continue
+	fi
+
+	shopt -s nullglob
+	for x in "${ED}"/etc/init.d/* ; do
+		if [[ -f ${x} ]] ; then
+			:
+		fi
+
+		if [[ $(head -n1 ${x}) =~ ^#!/sbin/runscript$ ]] ; then
```

Comment:
It's nicer to quote ${x} I think, even though nothing should contain spaces, it could in theory.